lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -154,6 +177,37 @@ printMemoryAccessAttribute(LoadStoreOpTy loadStoreOp, OpAsmPrinter *printer,
   elidedAttrs.push_back(spirv::attributeName<spirv::StorageClass>());
 }
 
+static LogicalResult verifyBinaryLogicalOp(Operation *logicalOp) {
```

Comment:
We don't need this manual checks anymore with `SameOperandsAndResultShape`.

Reply:
Fixed, thanks!